source/NVDAObjects/window/scintilla.py

```diff
@@ -66,6 +66,7 @@ class TextRangeStruct(ctypes.Structure):
 class ScintillaTextInfo(textInfos.offsets.OffsetsTextInfo):
 
 	def _getOffsetFromPoint(self,x,y):
+		x, y = winUser.ScreenToClient(self.obj.windowHandle, x, y)
 		return watchdog.cancellableSendMessage(self.obj.windowHandle,SCI_POSITIONFROMPOINT,x,y)
 
 	def _getPointFromOffset(self,offset):

```

Comment:
Could you please verify whether this method works as expected? Note that there is a long standing bug in this method (i.e. the `if point.x and point.y:` line). Please change this into `if point.x is not None and point.y is not None:`